packages/resilience/src/resilience.spec.ts

```diff
@@ -64,7 +65,7 @@ describe('combined resilience features', () => {
 
     expect(response.status).toBe(HTTP_STATUS.OK);
   }, 10000);
-  it('circuit breaker works together with a timeout', async () => {
+  it('uses circuit breaker and timeout with resilience()', async () => {
```

Comment:
[req] Perhaps you can add a few more tests with the retry being added in this file.